### ISSUE TITLE
Warn when downloading poor solutions

### ISSUE THREAD

**Issue by jeffreyhanson** [NCC-CNC/wheretowork#171]
@ricschuster mentioned the following point was made during a meeting: When a user wants to download a solution it would be good if the app could check whether criteria that are required were met (i.e. default values of the app for theme goals were not lowered and PA's were included). If the criteria were not met, a popup could show up generating a warning message. This would still allow the download, just warn the user that what they did can't be used for official business.

**Comment by ricschuster** [comment]
Agreed. No need to invest any time into this for this iteration of the tool.
